config/boards/radxa-zero3.csc

```diff
@@ -32,6 +43,21 @@ function post_family_config__radxa-zero3_use_vendor_uboot() {
 	}
 }
 
+function post_family_config_branch_current__radxa-zero3_use_mainline_uboot() {
```

Comment:
This is copy+pasted -- let me know if I should update the `edge` one as well, I don't really want to test it because I have to wire up the serial port again :)